libr/search/search.c

```diff
@@ -9,6 +9,12 @@
 
 R_LIB_VERSION (r_search);
 
+typedef struct {
+	ut64 end;
+	int len;
+	ut8 data[];
```

Comment:
use * instead of [] , it is more portable, especially for older compilers. Use [] only for fixed sizes